frontend/src/liboperator/operator/operator.go

```diff
@@ -556,17 +556,17 @@ func (w *wsIoWrapper) Read(buf []byte) (int, error) {
 		w.buf = readBuf
 		w.pos = 0
 	}
-	nRead := copy(buf[:], w.buf[w.pos:])
+	nRead := copy(buf, w.buf[w.pos:])
 	w.pos += nRead
 	return nRead, nil
 }
 
-func (w *wsIoWrapper) Write(buf []byte) (int, error) {
-	err := w.wsConn.WriteMessage(websocket.BinaryMessage, buf)
+func (w *wsIoWrapper) Write(p []byte) (int, error) {
```

Comment:
nit:  this name change is more relevant to the Read method: `func (w *wsIoWrapper) Read(buf []byte) (int, error) {`

Reply:
Changed argument name of the Read() function instead of this.